app/src/main.c

```diff
@@ -207,14 +211,15 @@ static SDL_bool parse_args(struct args *args, int argc, char *argv[]) {
         {"fullscreen",   no_argument,       NULL, 'f'},
         {"help",         no_argument,       NULL, 'h'},
         {"max-size",     required_argument, NULL, 'm'},
+        {"output",       required_argument, NULL, 'o'},
```

Comment:
In my opinion, the `output` is too general. In the future, we may add some feature like output format. It make a bit confusing.

Reply:
--output-file?